oci/defaults.go

```diff
@@ -4,33 +4,14 @@ import (
 	"os"
 	"runtime"
 
+	"github.com/docker/docker/oci/caps"
 	specs "github.com/opencontainers/runtime-spec/specs-go"
 )
 
 func iPtr(i int64) *int64        { return &i }
 func u32Ptr(i int64) *uint32     { u := uint32(i); return &u }
 func fmPtr(i int64) *os.FileMode { fm := os.FileMode(i); return &fm }
 
-// DefaultCapabilities returns a Linux kernel default capabilities
-func DefaultCapabilities() []string {
```

Comment:
@AkihiroSuda do you think we should make this an alias for the new location?

Reply:
I think either is fine